modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -62,7 +62,7 @@ class DrupalInstantArticleDisplay {
    *
    * @var InstantArticle
    */
-  private $instantArticle;
+  public $instantArticle;
```

Comment:
Ah ok. Yes. This was off my radar because I was still trying to get the entire object to be an instance of the fb SDK InstantArticle class. Yep def need to keep it this way till (if) we rework that in #35 :+1: